components/supervisor-api/generate.sh

```diff
@@ -53,7 +53,7 @@ local_java_protoc() {
         --java_out=java/src/main/java \
         ./*.proto
     # remove trailing spaces
-    find components/supervisor-api/java/src/main/java/io/gitpod/supervisor/api -maxdepth 1 -name "*.java" -exec sed -i -e "s/[[:space:]]*$//" {} \;
```

Comment:
Do we need this like anymore?

I had to comment it out, because it fails:

```
find: ‘components/supervisor-api/java/src/main/java/io/gitpod/supervisor/api’: No such file or directory
```